src/library/VTKExtensions/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# XXX not sure this is the right location
+# this is at least the deepest possible
+
+# Provide symbolic link to data in build
+# Required for testing
+if(BUILD_TESTING)
+  file(CREATE_LINK "${CMAKE_SOURCE_DIR}/testing/" "${CMAKE_CURRENT_BINARY_DIR}/Data" SYMBOLIC)
+endif()
+
+if(F3D_MODULE_RAYTRACING)
+  if(TARGET VTK::RenderingRayTracing)
+    message(STATUS "VTK RayTracing enabled")
+  else()
+    message(FATAL_ERROR "F3D_MODULE_RAYTRACING option is enabled but VTK::RenderingRayTracing is not found")
+  endif()
+endif()
+
+if(F3D_MODULE_ASSIMP)
```

Comment:
Assimp, OCCT and Alembic find_package should be in `src/library/VTKExtensions/Readers/CMakeLists.txt`

Reply:
Not sure to follow, what is vtkPVConfig?

Reply:
vtkF3DConfig.h sorry

Reply:
Let me be more specific here:

1. vtkF3DConfig is in Core module
2. vtkF3DConfig is responsible for cmakedefine for assimp and such
3. F3D_MODULE_ASSIMP can only be changed by the user, it is not transient
4. Implies we do not need to check for assimp existence before going into Core
5. But it can be confusing for developpers to use the F3D_MODULE_ASSIMP value before actually checking assimp

That is why I'm saying that technically you are right, but this may be a bad idea.

Reply:
Ok, I'm a bit confused, but I will follow your decision here. If you think it's better here, I'm fine.

Reply:
I put them back in main CMakeLists.txt anyway